app/backend/src/couchers/phone/sms.py

```diff
@@ -17,4 +20,33 @@ def send_sms(number, message):
     """Send SMS to a E.164 formatted phone number. Return "success" on
     success, "unsupported operator" on unsupported operator, and any other
     string for any other error."""
-    return "unsupported operator"
+
+    if not config["ENABLE_SMS"]:
+        logger.info(f"SMS not emabled, need to send to {number}: {message}")
+        return
+
+    if len(request.message) > 140:
+        return "message too long"
+
+    sns = boto3.client("sns")
+    sns.set_sms_attributes(
+        attributes={"DefaultSenderID": config.get("SMS_SENDER_ID"), "DefaultSMSType": "Transactional"}
```

Comment:
Hardcode DefaultSenderID to "Couchers.org", no need for a config variable.

Reply:
I guess you need some API key as config variable though.

Reply:
I think it's good to be able to change it e.g. on the "next" instance so one can disambiguate between the production deployment and staging

Reply:
Good point. The API key is needed though I guess.